.code-samples.meilisearch.yaml

```diff
@@ -3,10 +3,27 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+get_all_tasks_filtering_1: |-
+  client.tasks(index_uid: ['movies'])
```

Comment:
I chose to dot `client.index('movies').tasks in the PHP SDK, don't know if this is something you discussed, so just to let you know 😇